src/Webapi/Temporal/Webapi__Temporal__Difference.res

```diff
@@ -0,0 +1,22 @@
+module DifferenceOptions = {
```

Comment:
This submodule doesn't seem to add anything? I think it would work just as well without the submodule wrapper.